style.css

```diff
@@ -461,6 +469,7 @@ i.devider {
   justify-content: flex-start;
   list-style: none;
   padding-left: 0;
+  flex-wrap: wrap;
 }
 
 .work .tag {
```

Comment:
## Optional Suggestions 📌
---

- [x] Would look better if you add a more descriptive PR next time for example check out this [link ](https://github.com/saied2035/mobile-version-skeleton/pull/11l)  and that's how a descriptive PR should look like, normally a not descriptive PR is cause of invalidation as stated by Microverse Professionalism rules, so don't forget to implement it next time, for now, I will let it pass this time, your project looks excellent and all the functionalities were implemented, you did a very great job, congrats 👍 💯 

![tenor](https://user-images.githubusercontent.com/54015740/127129480-dfb12898-5429-4ff7-a67a-f4e3df0edc64.gif)

### Your project's been approved ✔😁♻ Keep working hard 💪✨✔